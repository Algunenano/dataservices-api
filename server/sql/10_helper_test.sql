-- Check that check_host is working
select check_host('cartodb.com');

-- Check that check_pwd is working
select pwd();
