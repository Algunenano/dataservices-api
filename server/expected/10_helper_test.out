-- Check that check_host is working
select check_host('cartodb.com');
 check_host 
------------
 t
(1 row)

-- Check that check_pwd is working
select pwd();
             pwd              
------------------------------
 /var/lib/postgresql/9.3/main
(1 row)

